includes/admin/class-sensei-onboarding.php

```diff
@@ -126,6 +129,91 @@ public function onboarding_page() {
 		<?php
 	}
 
+	/**
+	 * Check if current screen is selected to display the wizard notice.
+	 *
+	 * @return boolean
+	 */
+	private function is_current_screen_selected_to_wizard_notice() {
+		$screen = get_current_screen();
+
+		if ( false !== strpos( $screen->id, 'sensei-lms_page_sensei' ) ) {
+			return true;
+		}
+
+		$screens_without_sensei_prefix = [
+			'dashboard',
+			'plugins',
+			'edit-sensei_message',
+			'edit-course',
+			'edit-course-category',
+			'course_page_course-order',
+			'edit-module',
+			'course_page_module-order',
+			'edit-lesson',
+			'edit-lesson-tag',
+			'lesson_page_lesson-order',
+			'edit-question',
+			'question',
+			'edit-question-category',
+		];
+
+		return in_array( $screen->id, $screens_without_sensei_prefix, true );
+	}
+
+	/**
+	 * Onboarding wizard notice.
+	 *
+	 * @access private
+	 */
+	public function onboarding_wizard_notice() {
+		if (
+			! $this->is_current_screen_selected_to_wizard_notice()
+			|| ! get_option( self::SUGGEST_ONBOARDING_OPTION, 0 )
+			|| ! current_user_can( 'manage_sensei' )
+		) {
+			return;
+		}
+
+		$setup_url = admin_url( 'admin.php?page=' . $this->page_slug );
+
+		$skip_url = add_query_arg( 'sensei_skip_setup_wizard', '1' );
+		$skip_url = wp_nonce_url( $skip_url, 'sensei_skip_setup_wizard' );
+		?>
+		<div id="message" class="updated sensei-message sensei-connect">
+			<p><?php echo wp_kses_post( __( '<strong>Welcome to Sensei LMS</strong> &#8211; You\'re almost ready to start creating online courses!', 'sensei-lms' ) ); ?></p>
+
+			<p class="submit">
+				<a href="<?php echo esc_url( $setup_url ); ?>" class="button-primary">
+					<?php esc_html_e( 'Run the Setup Wizard', 'sensei-lms' ); ?>
```

Comment:
Hmm. @donnapep are we calling this the "Setup Wizard" externally? We seem to have both names mixed together in code (mentioned in another comment above). We should keep the naming consistent in-code, and maybe we should match what we are calling it externally as well?

Reply:
Yup, see https://github.com/Automattic/sensei/pull/3096#discussion_r421720947

Reply:
Renamed for this PR here: https://github.com/Automattic/sensei/pull/3096/commits/4782bd94094f308d1cdddbb1351de4bd72e82ccc